franklin/github/views.py

```diff
@@ -36,6 +38,7 @@ def get(self, request, format=None):
         site_serializer = FlatSiteSerializer(sites, many=True)
         return Response(site_serializer.data, status=status.HTTP_200_OK)
 
+    @validate_request_payload(['github', ])
     def post(self, request, format=None):
         project = request.data['github']
```

Comment:
you'll find `validate_request_payload` in helpers.py. Basically it validates that the POST contains a json object with the key `github` that is not empty. If so, it raises a nice uniform error. Thus, we can skip....

```
project = request.data.get('github', None)
if not project:
     raise
```

...and instead just know that the value is there `project = request.data['github']`